esmvaltool/recipes/hydrology/recipe_wflow.yml

```diff
@@ -53,5 +53,5 @@ diagnostics:
       script:
         script: hydrology/wflow.py
         basin: Meuse
-        dem_file: 'wflow/wflow_dem_Meuse.nc'
+        dem_file: 'wflow_parameterset/meuse/staticmaps/wflow_dem.map'
```

Comment:
Is this really needed for the ESMValTool shipped recipe? I guess the user has to set this anyway right?

Reply:
There two reasons for that: first, again the file format is more realistic in terms of application. The format of wflow dem files  are mentioned as `.map` and not `.nc` in [wflow documentation](https://wflow.readthedocs.io/en/latest/wflow_usage.html?highlight=wflow_dem.map#how-to-use-the-models). Second, this change makes it clear why we cannot move some of the regridding steps (e.g. [regridding pr variable in this line](https://github.com/ESMValGroup/ESMValTool/blob/64e5f574c9f9e45a15a4662417e59fa27468a99b/esmvaltool/diag_scripts/hydrology/wflow.py#L232)) from diagnostics to the recipe sections. Actually, esmvalcore regrid processor does not support a target grid in `.map` format.